modules/hilary/manifests/init.pp

```diff
@@ -147,7 +147,16 @@
       group   => $os_group,
       content => template('hilary/config.js.erb'),
       require => [ File[$upload_files_dir], File[$config_files_tmp_dir], File[$config_files_tmp_upload_dir] ]
```

Comment:
Need a ; at the end of this line to delimit from the added section as the closing brace is no longer immediately after.